packages/sparrowql/src/build.ts

```diff
@@ -44,20 +48,18 @@ export type Options = {
   query?: Record<string, unknown>;
   relations: Relation[];
   skip?: number;
-  sort?: any;
+  sort?: SortType;
   start: string;
   typeMap?: Record<string, string>;
 };
 
-type Step = {
-  group?: unknown;
-  limit?: number;
-  match?: unknown;
-  projection?: Record<string, unknown>;
-  relation?: Relation;
-  skip?: number;
-  sort?: Record<string, -1 | 1>;
-};
+type Step = {group: object }
+  | {limit: number }
+  | {match: object }
+  | {projection: ProjectionType }
+  | {relation: Relation }
+  | {skip: number }
+  | {sort: SortType };
```

Comment:
This seems weirdly formatted. Could you check it?

Reply:
Everything is still very WIP, only pushed the changes so I could see a clean diff. I will check it.